go-controller/pkg/egressFirewall/egressfirewall.go

```diff
@@ -1,13 +1,15 @@
-package ovn
+package egressfirewall
```

Comment:
Likewise, I think I'd prefer this to be in `pkg/ovn/egressfirewall`

Reply:
Also, the convention for package directory name would be `egressfirewall`, not `egressFirewall`

Reply:
done